hathor/stratum/stratum.py

```diff
@@ -523,8 +523,10 @@ def handle_submit(self, params: Dict, msgid: Optional[str]) -> None:
             tx.verify_pow()
         except PowError:
             # Transaction pow was not enough, but the share was succesfully submited
-            self.log.info('Insufficient weight, keep searching', tx=tx)
+            self.log.info('high hash, keep mining', tx=tx)
```

Comment:
The original message seems better to me ("insufficient weight" x "high hash").

Reply:
"high hash" is the expression that Bitcoin uses, and one problem with "insufficient weight" is that it was often mistaken for an error, that happened a lot at the beginning when new people were mining with their own fullnodes

Reply:
Got it